### PR TITLE
[documentation] Added information about database setup + removed unne…

### PR DESCRIPTION
**Pull request by Binary-Eater**
…cessary isCourseInstance relationship

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,21 @@ An app to track attendance at UIUC
...
+If setting up on an install of `postgresql` that has a `postgres` user, can simple execute `utils/db_setup/psqlInit.sh`
+
+### Manual Setup
+
+If not, simply execute `utils/db_setup/dbInit.sql` using the `psql` client.
+
+For example,
+
+```
+psql <normal-postgres-args> -a -f utils/db_setup/dbInit.sql
```
Comment (nwalters512) [inline README.md:25]: what are the normal postgres args? 😛

File: src/lib/config.js [hunk 2]
```diff
@@ -19,7 +19,6 @@ config.sqlInitFiles = [
     'section',
     'student',
     'isOwner',
-    'isCourseInstance',
```
Comment (nwalters512) [inline src/lib/config.js:22]: If we're going to be removing this, should also add the necessary config to the `course_instances` schema

File: README.md (hunk 1 above)
Comment (Binary-Eater) [reply README.md:25]: This varies due to the fact that users like @JParisFerrer need to do `-h localhost` due to WSL client to windows native host. I can put an example for the most common ones.

File: src/lib/config.js (hunk 2 above)
Comment (Binary-Eater) [reply src/lib/config.js:22]: Thought you already added a course key for some random reason.

File: README.md (hunk 1 above)
Comment (nwalters512) [reply README.md:25]: That'd be good - don't let people guess what those args might be!

Comment (Binary-Eater) [reply README.md:25]: Added that in the last commit of this PR.

**Review by nwalters512 — APPROVED** [review]

**Pull request merged**